### DISCUSSION TITLE
App ids can get very annoying...🤬

### DISCUSSION THREAD

**Discussion by Elbozo123** [SideStore/SideStore#603] [Ideas]
PLS ALLOW TO DELETE APP IDS!!!!!!🤬😡🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬👊🏻

**Comment by polymo1** [comment]
It's not possible to just "delete" app ids like that.

**Reply by Elbozo123** [reply]
But they dont ever get removed

**Reply by nythepegasus** [reply]
@Elbozo123 Pay Apple $99/year then you can delete them from https://developer.apple.com otherwise you have to wait for free App IDs to expire.
